dev/scripts/src/alluxio.org/command/command.go

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
This is a copy of the one in the private repo. Will this be replaced by an import eventually?

Reply:
not in the near future. if there becomes significant overlap, then we'd consider doing imports but the coupling is too weak at the moment to incur the overhead of establishing a dependency relationship.